### PR TITLE
Add LesionCountWeightedByAssignment, RefLesionsCount, PredLesionsCount

### LINKED ISSUE #4: add lesion counts to the package and output in CSV

### PR DESCRIPTION
**Pull request by valosekj**
@naga-karthik lmk once the PR is ready for review

Resolves #4

### CODE REVIEW

**Comment by naga-karthik** [comment]
hey @valosekj it is now ready for review! (for some reason, I am unable to add you as a reviewer)

**Comment by valosekj** [comment]
Thanks @naga-karthik! Tested and LGTM! I have just fixed a mission comma in https://github.com/ivadomed/MetricsReloaded/pull/6/commits/7f413e8a5881fe060f31cac9532b39e15811fc77. And clarified some comments in my other commits.

File: compute_metrics_reloaded.py [hunk 1]
```diff
@@ -246,9 +252,9 @@ def main():
     df_mean = (df.drop(columns=['reference', 'prediction', 'EmptyRef', 'EmptyPred']).groupby('label').
                agg(['mean', 'std']).reset_index())
 
-    # Rename columns
-    df.rename(columns={metric: METRICS_TO_NAME[metric] for metric in METRICS_TO_NAME}, inplace=True)
-    df_mean.rename(columns={metric: METRICS_TO_NAME[metric] for metric in METRICS_TO_NAME}, inplace=True)
+    # # Rename columns
+    # df.rename(columns={metric: METRICS_TO_NAME[metric] for metric in METRICS_TO_NAME}, inplace=True)
+    # df_mean.rename(columns={metric: METRICS_TO_NAME[metric] for metric in METRICS_TO_NAME}, inplace=True)
```
Comment (valosekj) [inline compute_metrics_reloaded.py:257]: @naga-karthik, I made all the metrics camel case in https://github.com/ivadomed/MetricsReloaded/pull/6/commits/0b1894a25d1909036a8ec74bcc3cf617ed830026. So maybe we can again use this dict?

Comment (naga-karthik) [reply compute_metrics_reloaded.py:257]: thanks for this! looks good now

**Comment by naga-karthik** [comment]
looks good ; merging!

**Pull request merged**